package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.6.14",
+  "version": "0.7.0",
```

Comment:
Better late than never (since we have now released Alterations).

Thoughts? Should this be 1.0.0? I didn't want to be presumptious.